src/Sylius/Component/User/Security/UserPasswordHasherInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\User\Security;
+
+use Sylius\Component\User\Model\CredentialsHolderInterface;
+
+interface UserPasswordHasherInterface
```

Comment:
Instead of creating the second interface, can't we just use our previous interface as ACL and just replace the implementation?